File: clients/csharp/examples/tables-list-type-names.cs
using System.Text.Json;
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Get an existing table
    var client = new DataAPIClient();
    var database = client.GetDatabase(
      "**API_ENDPOINT**",
      "**APPLICATION_TOKEN**"
    );

    // List type names
    var names = await database.ListTypeNamesAsync();

    Console.WriteLine(JsonSerializer.Serialize(names));
  }
}

File: clients/java/examples/tables-list-type-names.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;
import java.util.List;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // List type names
    List<String> result = database.listTypeNames();

    System.out.println(result);
  }
}
